docs/03-pages/01-building-your-application/06-configuring/14-preview-mode.mdx

```diff
@@ -219,6 +219,8 @@ You can pass an object to `setPreviewData` and have it be available in `getStati
 
 The preview mode works on `getServerSideProps` as well. It will also be available on the `context` object containing `preview` and `previewData`.
 
+> **Good to know**: Avoid setting the `Cache-Control` header because it cannot be bypassed. Instead, use [ISR](/docs/pages/building-your-application/data-fetching/incremental-static-regeneration).
```

Comment:
```suggestion
> **Good to know**: You shouldn't set the `Cache-Control` header when using Preview Mode because it cannot be bypassed. Instead, we recommend using [ISR](/docs/pages/building-your-application/data-fetching/incremental-static-regeneration).
```